### PR TITLE
#11 Add content for About page

### LINKED ISSUE #11: Add content for About page

### PR DESCRIPTION
**Pull request by VKTB**
Closes #11 

What it says in the title

### CODE REVIEW

File: content/about/index.md [hunk 1]
```diff
@@ -2,4 +2,19 @@
 title: About
 ---
 
-Example about page (index)
+The ICAT software takes data from large scientific facilities - such as particle accelerators - and catalogues and indexes it so scientists can find the data they need and share it with their team. A simple REST interface allows scientific instrument designers to quickly upload their data to the system for indexing and cataloging. TopCat, ICAT's easy to use web interface, allows scientists and collaborators to effortlessly search and download their data. ICAT's secure authentication system allows principal investigators to control their data and ensures users can only access data which they are allowed to access. (Mention DOIs here?)
+
+ICAT is in use at some of the world's largest scientific facilities (list some?) enabling access to millions of (pounds or dollars) worth of scientific results. The software is free & open-source and has been developed & maintained by an international collaboration for over 10 years.
+
+### What does ICAT do?
+For a more detailed explanation of what ICAT does and the actors, entities and workflows it models, see the [ICAT Model](about/icat_model.md) page.
```
Comment (VKTB) [inline content/about/index.md:10]: The hyperlink leads to a page that does not exist at the moment, shall I get rid of this paragraph and put it back when the page is created?

File: content/about/index.md [hunk 2]
```diff
@@ -2,4 +2,19 @@
...
+
+### Who uses ICAT?
+ICAT is used at many large scientific facilities. To find out how ICAT is supporting great science, see our case studies:
+- [CLF](collaboration/facilities/clf/)
+- [HZB](collaboration/facilities/hzb)
+- [ISIS](collaboration/facilities/isis/)
+- [OCTOPUS](collaboration/facilities/octopus/)
+
+### Who develops ICAT?
+ICAT is developed by an international collaboration. (TODO: list them/link to them). There are monthly meetings to discuss development, yearly face-to-face meetings and a steering committee which agrees the roadmap. To find out more, see our [Collaboration](/collaboration/) page.
```
Comment (VKTB) [inline content/about/index.md:20]: I can address the TODO here if we know and have the links to them, otherwise I will open an issue and address it later.

Comment (agbeltran) [reply content/about/index.md:20]: Please, remove the TODO. All the information about the collaboration can be added at https://icatproject.org/collaboration

**Review by agbeltran — CHANGES_REQUESTED** [review]
See changes suggested in the specific comments.

**Pull request merged**